eZ/Publish/Core/Persistence/Legacy/Content/Mapper.php

```diff
@@ -16,15 +16,50 @@
 use eZ\Publish\SPI\Persistence\Content\Language\Handler as LanguageHandler;
 use eZ\Publish\SPI\Persistence\Content\Relation;
 use eZ\Publish\SPI\Persistence\Content\Relation\CreateStruct as RelationCreateStruct;
+use eZ\Publish\SPI\Persistence\Content\Type\Handler as ContentTypeHandler;
 use eZ\Publish\SPI\Persistence\Content\VersionInfo;
+use Ibexa\Contracts\Core\Event\Mapper\ResolveMissingFieldEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
 
 /**
  * Mapper for Content Handler.
  *
  * Performs mapping of Content objects.
+ *
+ * @phpstan-type TVersionedLanguageFieldDefinitionsMap array<
+       int, array<
+           int, array<
+               string, array<
+                   int, \eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition,
+               >
+           >
+       >
+   >
+ * @phpstan-type TVersionedFieldMap array<
+     int, array<
+          int, array<
+              int, \eZ\Publish\SPI\Persistence\Content\Field,
+          >
+      >
+  >
+ * @phpstan-type TVersionedNameMap array<
+     int, array<
+         int, array<
+             string, array<int, string>
+         >
+     >
+ >
+ * @phpstan-type TContentInfoMap array<int, \eZ\Publish\SPI\Persistence\Content\ContentInfo>
+ * @phpstan-type TVersionInfoMap array<
+     int, array<
+         int, \eZ\Publish\SPI\Persistence\Content\VersionInfo,
+     >
+ >
```

Comment:
Malformed PHPDoc block
```suggestion
 *
 * @phpstan-type TVersionedLanguageFieldDefinitionsMap array<
 *     int, array<
 *         int, array<
 *             string, array<
 *                 int, \eZ\Publish\SPI\Persistence\Content\Type\FieldDefinition,
 *             >
 *         >
 *     >
 * >
 * @phpstan-type TVersionedFieldMap array<
 *   int, array<
 *        int, array<
 *            int, \eZ\Publish\SPI\Persistence\Content\Field,
 *        >
 *    >
 * >
 * @phpstan-type TVersionedNameMap array<
 *   int, array<
 *       int, array<
 *           string, array<int, string>
 *       >
 *   >
 * >
 * @phpstan-type TContentInfoMap array<int, \eZ\Publish\SPI\Persistence\Content\ContentInfo>
 * @phpstan-type TVersionInfoMap array<
 *   int, array<
 *       int, \eZ\Publish\SPI\Persistence\Content\VersionInfo,
 *   >
 * >
```